app/controllers/application_controller.rb

```diff
@@ -22,12 +23,11 @@ def append_to_log(params)
   # Rails' instrumentation code, and is run after each request.
   def append_info_to_payload(payload)
     super
-    if @custom_log_items
-      payload[:custom_log_items] = @custom_log_items
-    end
-  end
 
-  # :nocov:
+    append_to_log(request_id: RequestStore.store[:request_id])
```

Comment:
To me it made sense, the other option is to have it in a before action, which didn't seem right. Since this method is part of Rails instrumentation it seemed right to put it here. Any suggestions?